x/ref/lib/vdl/build/build.go

```diff
@@ -311,6 +312,70 @@ func SrcDirs(errs *vdlutil.Errors) []string {
 	return append(srcDirs, vdlPathSrcDirs(errs)...)
 }
 
+// HasGoModule returns the value of the module statement in the go.mod file in
+// the directory specified by path, or an empty string otherwise.
+func HasGoModule(path string) string {
+	buf, err := ioutil.ReadFile(filepath.Join(path, "go.mod"))
+	path = filepath.Dir(path)
+	if err != nil && os.IsNotExist(err) {
```

Comment:
What if the error is of some other kind?

Reply:
fixed to return an error.